.travis.yml

```diff
@@ -11,6 +13,7 @@ cache:
 stages:
   - lint
   - test
+  - lint macOS
```

Comment:
you can use the same stage `lint` - it will have this stage with two jobs, one in linux, one in macOS